src/sentry/lang/javascript/processor.py

```diff
@@ -697,6 +698,9 @@ def process_frame(self, processable_frame, processing_task):
                     # (i.e. node_modules)
                     if "/~/" in filename:
                         filename = "~/" + abs_path.split("/~/", 1)[-1]
+                    elif WEBPACK_WITH_LIBRARY_PREFIX_RE.search(filename):
+                        # https://github.com/getsentry/sentry/issues/20745
+                        filename = WEBPACK_WITH_LIBRARY_PREFIX_RE.search(filename).group(1)
```

Comment:
We can skip running regexp twice with:
```js
webpack_prefixed = WEBPACK_WITH_LIBRARY_PREFIX_RE.search(filename)
if webpack_prefixed:
    filename = webpack_prefixed.group(1)
```

Reply:
Adjusted to code so it'll run the regex once, thanks